src/main/java/mods/railcraft/common/plugins/jei/rockcrusher/RockCrusherMachineCategory.java

```diff
@@ -37,10 +39,10 @@
     private final IDrawable progress;
 
     public RockCrusherMachineCategory(IGuiHelper guiHelper) {
-        ResourceLocation location =  new ResourceLocation(RailcraftConstants.GUI_TEXTURE_FOLDER + "gui_crusher.png");
-        background = guiHelper.createDrawable(location, 16, 20, width, height);
+        ResourceLocation location = new ResourceLocation(RailcraftConstants.GUI_TEXTURE_FOLDER + "gui_crusher.png");
```

Comment:
This needs to be replaced. The charge bar is probably useless in this case. I can supply the older GUI texture that can be designated for usage in JEI.